pom.xml

```diff
@@ -376,6 +377,28 @@
         </executions>
       </plugin>
 
+      <!-- Enforce versions -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>1.4</version>
+        <executions>
+          <execution>
+            <id>enforce-versions</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireJavaVersion>
+                  <version>${java.version}</version>
```

Comment:
oops, I was still right. here is the doc about the version syntax:
https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html